routes/recipesRouter.js

```diff
@@ -0,0 +1,37 @@
+import express from 'express';
+import authenticate from '../middlewares/authenticate.js';
+
+import listRecipes from '../controllers/recipes/listRecipes.js';
+import getRecipeById from '../controllers/recipes/getRecipeById.js';
+import getPopularRecipes from '../controllers/recipes/getPopularRecipes.js';
+
+import createRecipe from '../controllers/recipes/createRecipe.js';
+import deleteRecipe from '../controllers/recipes/deleteRecipe.js';
+import getMyRecipes from '../controllers/recipes/getMyRecipes.js';
+
+import addToFavorites from '../controllers/recipes/addToFavorites.js';
+import removeFromFavorites from '../controllers/recipes/removeFromFavorites.js';
+import getFavoriteRecipes from '../controllers/recipes/getFavoriteRecipes.js';
+
+import validateBody from '../helpers/validateBody.js';
+import recipeSchema from '../schemas/recipes/createRecipeSchema.js';
+
+const recipesRouter = express.Router();
+
+// Public routes
+recipesRouter.get('/list', listRecipes);
```

Comment:
```suggestion
recipesRouter.get('/', listRecipes);
```
To be in tact with other get endpoints(e.g. /categories, /areas, /ingredients)